app/res/values/strings.xml

```diff
@@ -433,4 +433,6 @@
     <string name="location_provider_disabled" cc:translatable="true">Turn on your location to receive location updates</string>
     <string name="wait_for_location_fix"  cc:translatable="true">CommCare is still trying to get your location. Please wait or click Cancel to abort.</string>
     <string name="invalid_case_property_length" cc:translatable="true">Invalid %1s, value must be 255 characters or less</string>
+    <string name="root_detected_title" cc:translatable="true">Rooted Device Detected</string>
```

Comment:
Unrelatedly, how do we decide whether to put the new strings in strings.xml or android_translatable_strings.txt file?

Reply:
My preference is `strings.xml` for anything new since it provides IDE support. we traditionally seems to have used `android_translatable_strings.

Reply:
Got it. Thanks!